P5/Exemplars/tei_rdfa.odd

```diff
@@ -0,0 +1,227 @@
+          class `att.global.analytic.rdfa`. We argue that these tagdocs elements
+          are not going to be used in the context where RDFa support is relevant
+        -->
+
+        <classSpec type="atts" ident="att.global.analytic.rdfa" mode="add">
+          <desc xml:lang="en" versionDate="2020-07-01">provides additional global attributes for
+            embedding RDF following the RDFa Core 1.1 specification</desc>
+          <attList>
+            <attDef mode="add" ident="about">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-about</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a SafeCURIEorCURIEorIRI, used for stating
+                what the data is about (a 'subject' in RDF terminology)</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="content">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-content</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a CDATA string, for supplying
+                machine-readable content for a literal (a 'literal object', in RDF
+                terminology)</desc>
+              <datatype>
+                <dataRef key="teidata.text"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="datatype">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-datatype</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a TERMorCURIEorAbsIRI representing a
+                datatype, to express the datatype of a literal</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="href">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-href</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">a traditionally navigable IRI for
+                expressing the partner resource of a relationship (a 'resource object', in RDF
+                terminology)</desc>
+              <datatype>
+                <dataRef key="teidata.enumerated"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="inlist">
+              <gloss xml:lang="en" versionDate="2020-07-01">rdfa-inlist</gloss>
+              <desc xml:lang="en" versionDate="2020-07-01">An attribute used to indicate that the
+                object associated with a rel or property attribute on the same element is to be
+                added to the list for that predicate. The value of this attribute MUST be ignored.
+                Presence of this attribute causes a list to be created if it does not already
+                exist.</desc>
+              <datatype>
+                <dataRef key="teidata.nullOrName"/>
+              </datatype>
+            </attDef>
+            <attDef mode="add" ident="prefix">
```

Comment:
`@prefix` probably isn't needed in a TEI context since we can use `@xmlns` to declare prefixes. Non-XHTML HTML can't, hence the need for the attribute.